test/loadTest/orion_updates_ngsiv2.jmx

```diff
@@ -0,0 +1,719 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I think that in the repo should to store the script with the listener to get metrics up, and if someone would like to disable it is very easy, in GUI or in file directly (change enabled="false"). Avoid scripts duplicate.
```
<kg.apc.jmeter.perfmon.PerfMonCollector guiclass="kg.apc.jmeter.vizualizers.PerfMonGui" testclass="kg.apc.jmeter.perfmon.PerfMonCollector" testname="PerfMon Metrics Collector" enabled="true">
```